docs/architecture/decisions/0006-aws-vpc-public-and-private-subnets.md

```diff
@@ -0,0 +1,29 @@
+# 6. AWS VPC Public and Private subnets
+
+Date: 2020-04-20
+
+## Status
+
+Accepted
+
+Amends [5. VPC and Network Design](0005-vpc-and-network-design.md)
+
+## Context
+
+Mozilla IT SREs have built a VPC module at https://github.com/mozilla-it/itsre-deploy/tree/master/modules/vpc. This module does almost everything described in our previous ADR (#5), with one exception around public/private subnets. That difference will be described in this document.
+
+## Decision
+
+We will create two subnets per az. One 'public' and one 'private'. In general this is mostly by convention, rather than something that is strictly enforced.  We may later validate instances in private are not accessible to the general public with automated auditing solution. But, for now, we'll just do our best to only assign public things to the public subnet, and everything else can go in private.
```

Comment:
are subnets tagged public/private?

Reply:
name has public/private in it. but there doesn't look to be a separate tag.  If we want that (for whatever reason) we'd need to put a pr in against the IT-SE terraform repo

Reply:
name w/ public/private should be ok, thanks!